main.go

```diff
@@ -152,6 +854,51 @@ var (
 			snapshotKey: "master/tasks_starting",
 			valueType:   prometheus.GaugeValue,
 		},
+		snapshotMetric{
+			desc: prometheus.NewDesc(
+				"mesos_master_uptime_secs",
+				"Mesos master uptime in seconds.",
+				masterMetricsLabels, nil,
+			),
+			snapshotKey: "master/uptime_secs",
+			valueType:   prometheus.CounterValue,
```

Comment:
This is actually a Gauge (you care about the absolute value, not how fast it's increasing)